src/main/java/it/pagopa/transactions/client/NodeForPspClient.java

```diff
@@ -32,6 +32,12 @@ public class NodeForPspClient {
     @Value("${nodo.nodeforpsp.uri}")
     private String nodoPerPspUri;
 
+    @Value("${nodo.nodeforpsp.apikey}")
```

Comment:
Maybe here we can refactor this class to use constructor injection? what do you think? this way we can avoid reflection to set those api key into test class

Reply:
right, but better to do it in a dedicated PR